pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/ThinPexGenerator.java

```diff
@@ -44,7 +44,7 @@
     public ThinPexGenerator(Project project, String entryPointTemplate, Map<String, String> extraProperties) {
         this.project = project;
         this.entryPointTemplate = entryPointTemplate;
-        this.extraProperties = extraProperties == null ? new HashMap<>() : extraProperties;
+        this.extraProperties = extraProperties == null ? new HashMap<String, String>() : extraProperties;
```

Comment:
No. Leave it as is then. It seems that Java 8 improved even the diamond inference. That's why I preferred that we use Java 8. Oh, well ... :-)